pkg/apis/kops/v1alpha2/componentconfig.go

```diff
@@ -610,12 +610,6 @@ type KubeSchedulerConfig struct {
 	// which has been supported as far back as Kubernetes 1.7. The default depends on the version and the cloud provider
 	// as outlined: https://kubernetes.io/docs/concepts/storage/storage-limits/
 	MaxPersistentVolumes *int32 `json:"maxPersistentVolumes,omitempty"`
-	// Qps sets the maximum qps to send to apiserver after the burst quota is exhausted
-	Qps *resource.Quantity `json:"qps,omitempty" configfile:"ClientConnection.QPS"`
-	// Burst sets the maximum qps to send to apiserver after the burst quota is exhausted
-	Burst *int32 `json:"burst,omitempty" configfile:"ClientConnection.Burst"`
```

Comment:
How is the admin going to configure `Qps` and `Burst`? I would think you would need these two fields, but without the `configfile` tags.

Reply:
They are deprecated as flags already, the goal of this PR was to add the alternative of using the --config option. I took it out of v1alpha[1|2] as per previous comment.

Reply:
The previous comment was to remove the `configfile` struct tags, not the fields. There was also a comment questioning the need for the `KubeConfig` flag.

Right now I see no way for the admin to configure the `Qps` and `Burst` settings in the `kops.KubeSchedulerConfig` struct. If they can't be configured, what is their purpose?

Reply:
But what's the point of having the field without the configfile tag since they are not going to be rendered? It's true that the fields shouldn't have been removed though, so can you explain why the tag should be removed? Like the FeatureGates or LeaderElection flags are all there for all alphas.

Reply:
The `v1alpha1` or `v1alpha2` struct is what gets parsed from the versioned resource. That gets converted to the `kops` struct. It is the `kops` struct that gets rendered, not the `v1alpha1` or `v1alpha2` struct.

Reply:
Hmm, nice, Thanks, It confused me that all the flags were there, I'll send the update in a few